generated/nimodinst/unit_tests/matchers.py

```diff
@@ -4,6 +4,9 @@
 
```

Comment:
nimodinst.Session didn't get this.

Reply:
Added __repr__ and __str__ to nimodinst Session and Device.
```python
s = nimodinst.Session('')
s
print(s)
```
Results in this output:
```
>>> s = nimodinst.Session('')
>>> s
nimodinst.Session(driver='')
>>> print(s)
nimodinst.Session(driver=''):
nimodinst.Device(owner=nimodinst.Session(driver=''), index=0):
    bus_number = 0
    chassis_number = 0
    device_model = u'NI USB-5133'
    device_name = u'Dev1'
    max_pciexpress_link_width = -1
    pciexpress_link_width = -1
    serial_number = u'00EB9293'
    slot_number = 0
    socket_number = 0

nimodinst.Device(owner=nimodinst.Session(driver=''), index=1):
    bus_number = 0
    chassis_number = 0
    device_model = u'USB-4065'
    device_name = u'Dev2'
    max_pciexpress_link_width = -1
    pciexpress_link_width = -1
    serial_number = u'00E950F2'
    slot_number = 0
    socket_number = 0

```